guides/common/assembly_monitoring-hosts-using-red-hat-insights.adoc

```diff
@@ -1,5 +1,9 @@
 include::modules/con_monitoring-hosts-using-red-hat-insights.adoc[]
 
+ifdef::orcharhino[]
```

Comment:
We also expose it in the `katello` installer scenario so I think we should also expose it here. @theforeman/rh-cloud?

Reply:
This is an old plugin, effectively replaced by [foreman_rh_cloud](https://github.com/theforeman/foreman_rh_cloud) in Satellite. Do we still need the documentation for it?

Reply:
I believe this plugin is installed automatically with Satellite and optional on orcharhino. I am unsure _if_ we should show this procedure for Katello too. I am fine either way.

Reply:
I didn't look at the titles. The installer command does install foreman_rh_cloud.

@maximiliankolb can you rename it and call it by the proper name?

Reply:
Is there a difference between "Red Hat Insights" plugin and "foreman_rh_cloud"? I am not sure what you're expecting here; the assembly is talking about RH Insights.

Reply:
Red Hat insights plugin is https://github.com/redhataccess/foreman-plugin and deprecated. The installer command you use installs https://github.com/theforeman/foreman_rh_cloud. It's correct that you want to install the non-deprecated plugin, but the naming should also be correct.

Reply:
Still unsure what I should rename:

* https://docs.theforeman.org/nightly/Managing_Hosts/index-satellite.html#Monitoring_Hosts_Using_Red_Hat_Insights_managing-hosts
* https://github.com/theforeman/foreman-documentation/blob/master/guides/common/assembly_monitoring-hosts-using-red-hat-insights.adoc?plain=1
* https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_monitoring-hosts-using-red-hat-insights.adoc?plain=1

Tested on Foreman 3.5 and 3.7:
````
$ foreman-installer --full-help
...
    --[no-]enable-foreman-plugin-rh-cloud                                        Enable 'foreman_plugin_rh_cloud' puppet module (default: false)
...
````

Am I supposed to use `{RHCloud}` from https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L117?

Reply:
@ShimShtein perhaps you can elaborate.

Reply:
> https://docs.theforeman.org/nightly/Managing_Hosts/index-satellite.html#Monitoring_Hosts_Using_Red_Hat_Insights_managing-hosts

This one is correct, it's about using `insghts-client` and I see it also describes the functionality exposed by the rh_cloud plugin.

As for usage of rh_cloud as opposed to Insights: Insights is a general brand, used to describe all the cloud features we have on console.redhat.com. rh_cloud is the plugin that exposes the data from those features on Foreman (Satellite) UI. Specifically we expose the recommendations and remediations parts of the Insights. So I suppose the `{RHCloud}` macro is relevant for the installation options, but in the documentation of the functionality we should still use Insights.